source/common/unified_malloc_framework/src/memory_tracker.cpp

```diff
@@ -84,11 +88,30 @@ umfMemoryTrackerRemove(umf_memory_tracker_handle_t hTracker, const void *ptr,
 
 extern "C" {
 
-umf_memory_tracker_handle_t umfMemoryTrackerGet(void) {
-    static umf_memory_tracker_t tracker;
-    return &tracker;
+#if defined(_WIN32) && defined(UMF_SHARED_LIBRARY)
```

Comment:
it's fine for now while we have only one global, but eventually we might want to consolidate this somehow and create windows/linux abstractions (like in the loader).